project-base/storefront/components/Pages/ProductDetail/ProductDetailTabs/ProductDetailTabs.tsx

```diff
@@ -40,30 +40,78 @@ export const ProductDetailTabs: FC<ProductDetailTabsProps> = ({ description, par
 
             {!!parameters.length && (
                 <TabsContent headingTextMobile={t('Parameters')}>
-                    <Table className="border-0 p-0 max-w-screen-lg mx-auto">
-                        {parameters.map((parameter) => (
-                            <Row
-                                key={parameter.uuid}
-                                className="bg-tableBackground odd:bg-tableBackgroundContrast border-none"
-                            >
-                                <Cell className="py-2 text-left text-sm font-bold uppercase leading-5">
-                                    {parameter.name}
-                                </Cell>
+                    {parameters
+                        .filter(parameter => parameter.group === null)
+                        .sort((a, b) => a.name.localeCompare(b.name))
+                        .length > 0 && (
```

Comment:
Please avoid complex logic inside the `return` statement. I would extract this above the `return` and give it a descriptive name such as `sortedIndividualParameters`. This would also help optimise the code as you are doing the same filtering and sorting below again to map each parameter to `Row`.